app/controllers/api/v1/keywords_controller.rb

```diff
@@ -3,10 +3,12 @@
 module Api
   module V1
     class KeywordsController < ApplicationController
+      include Pagy::Backend
```

Comment:
```suggestion
      include Pagy::Backend

```

Reply:
If you agree with Thanh's suggestion, it can be interesting to move even more into the Application Controller:
- `def pagination_params`
- `def meta_from_pagy(pagy)`
- and add a new method `paginated_authorized(klass)` that will return `pagy(policy_scope(klass), pagination_params)`
- then you can also make the `pagination_params` method as `private`.

Reply:
Thanks for the very detailed suggestion; much appreciated

Reply:
solved in fe2c3fd